core/src/main/java/hudson/link/ConsoleURLProvider.java

```diff
@@ -0,0 +1,57 @@
+package hudson.link;
+
+import hudson.ExtensionList;
+import hudson.ExtensionPoint;
+import hudson.Util;
+import hudson.model.Run;
+import jenkins.model.Jenkins;
+
+/**
+ * Extension point for providing console urls
+ * @since 2.417
```

Comment:
I think the process is to leave these blank but in place for now, and then if/when this gets merged, someone will come back later after it gets released and update them.

```suggestion
 * @since TODO
```